### PR TITLE
219 move across wiki

### LINKED ISSUE #219: Move documentation to Sphinx
Move the docs from the GitHub wiki to Sphinx. 
- [x] Set up sphinx, with placeholders for user and developer documentation
- [x] Set up to compile and display on readthedocs.com
- [ ] Move across the existing documentation to the new format

### PR DESCRIPTION
**Pull request by tyronerees**
#### Description of Work

Fixes #219 

Information on the wiki has been moved across to sphinx.  Note: information has been moved across as-is, and may still be out of date

#### Testing Instructions

1. run `make html` in the docs directory
2. check that outputted files are formatted correctly, links work, image display, etc.
3. check that there's nothing in the wiki that's missing from the sphinx docs

### CODE REVIEW

File: docs/source/users/getting_started.rst [hunk 1]
```diff
@@ -1,5 +1,193 @@
+.. _getting-started:
+
+###############
 Getting Started
-===============
+###############
+
+|Python 2.7| will be needed for running/installing this. Instructions
+for setting up python are available
+`here <https://github.com/mantidproject/fitbenchmarking/wiki/Setting-up-Python>`__.
```
Comment (AndrewLister-STFC) [inline docs/source/users/getting_started.rst]: This link is redundant, the linked page is now copied below

File: docs/source/users/getting_started.rst [hunk 2]
```diff
@@ -1,5 +1,193 @@
...
+
+1. Download this repository or clone it using
+   `git <https://git-scm.com/>`__:
+   ``git clone https://github.com/mantidproject/fitbenchmarking.git``
+2. Open up a terminal (command prompt) and go into the
+   ``fitbenchmarking`` directory.
+3. Once you are in the right directory, type
+   ``python setup.py install``.
+4. Install mantid, platform specific instructions
+   `here <https://github.com/mantidproject/fitbenchmarking/wiki/Installing-Mantid>`__.
```
Comment (AndrewLister-STFC) [inline docs/source/users/getting_started.rst]: Same as above. Linked page is now in this file.

File: docs/source/users/getting_started.rst [hunk 3]
```diff
@@ -1,5 +1,193 @@
...
+5. Finally run ``fitbenchmarking/example_scripts/example_runScript.py``,
+   located in the example\_scripts folder. This example script fit
+   benchmarks Mantid using all the available minimizers. The result
+   tables can be found in ``example_scripts/results``.
+
+.. |Python 2.7| image:: https://img.shields.io/badge/python-2.7-blue.svg
+   :target: https://www.python.org/downloads/release/python-2715/
+
+******************
+Settting up Python
```
Comment (AndrewLister-STFC) [inline docs/source/users/getting_started.rst]:
```suggestion
Setting up Python
```

File: docs/source/users/index.rst [hunk 4]
```diff
@@ -8,5 +8,5 @@ Welcome to the documentation for the FitBenchmarking software.
     :caption: Contents:
 
     getting_started
-    examples
+    run_scripts
```
Comment (AndrewLister-STFC) [inline docs/source/users/index.rst]: This is just a comment but I think "examples" is clearer. Perhaps that just falls under the next step of sorting this out once its copied (e.g. a different PR)

File: docs/source/users/getting_started.rst (hunk 1 above)
Comment (tyronerees) [reply docs/source/users/getting_started.rst]: Good spot -- thanks! Fixed

File: docs/source/users/getting_started.rst (hunk 2 above)
Comment (tyronerees) [reply docs/source/users/getting_started.rst]: Also fixed

File: docs/source/users/index.rst (hunk 4 above)
Comment (tyronerees) [reply docs/source/users/index.rst]: I think you're right, so I've just added this PR, while we spot it.

**Pull request merged**